docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -5,12 +5,7 @@ weight: 300
 
 # Tempo in Grafana
 
-[Grafana 7.4](https://grafana.com/grafana/download/7.4.5) and later have a built in Tempo datasource that can be used to query Tempo and visualize traces.  This page describes the high-level features and their availability.  Use the latest versions for best compatibility and stability.
-
-1. [View trace by ID](#view-trace-by-id)
-2. [Log Search](#log-search)
-3. [Tempo Search](#tempo-search)
-4. [View JSON file](#view-json-file)
+[Grafana 7.4](https://grafana.com/grafana/download/7.4.5) and more recent versions have a built-in Tempo datasource that can be used to query Tempo and visualize traces.  This page describes the high-level features and their availability.  Use the latest versions for best compatibility and stability.
```

Comment:
Do we want to update the version we point to here?

Reply:
I think it's correct to document here that 7.4.5 is the first released version of Grafana with Tempo support. Further down we reference versions required for other features. Unsure if we should consolidate this somewhere or if it's fine as is.

Will defer to @KMiller-Grafana.

Reply:
I was thinking we could still mention that this release is the first to support the feature, but perhaps we want to encourage later versions.

Reply:
I assume that v7.4 is the earliest version that can use the feature.  I wouldn't advance that number, as it would be incorrect, and it would suggest to customers an unneeded upgrade of their Grafana instance.

I don't think we need to consolidate anything. The 2 different Tempo search methods each specify what they need (as a minimum version of Grafana).  I wouldn't want readers to have to look elsewhere for this info.

Reply:
I'm thinking more about the implications of pointing users to older versions of Grafana.  For example, I'm seeing a few vulnerabilities here that mention 7.4.5 specificlally.

https://www.cvedetails.com/vulnerability-list/vendor_id-18548/Grafana.html

Too, this is the first place we point users at to download in this page.  Further down we mention 8.x, but no download link etc.  Do you think it would be reasonable to mention what exists currently, but perhaps include a link to later versions so users reading this page could grab a more recent (and perhaps recommended) version?  Perhaps I'm overthinking it.

Reply:
Ah.  It is the link that is objectionable!  I'll just remove the link, but keep the info on the minimum version that can be used to have access to the feature.